templates/journeys/todos.journey.ts

```diff
@@ -0,0 +1,18 @@
+import { journey, step, monitor, expect } from '@elastic/synthetics';
+
+journey('TodoMVC Synthetics', ({ page, params }) => {
+  // Only relevant for the push command to create
+  // monitors in Kibana
+  monitor.use({
+    id: 'todomvc',
+    schedule: 10,
+  });
+  step('launch application', async () => {
+    await page.goto(params.url);
+  });
+
+  step('assert title', async () => {
+    const header = await page.$('h1');
+    expect(await header.textContent()).toBe('todos');
+  });
+});
```

Comment:
IMO this could be a simpler example against a live webpage.

Reply:
Feel free to suggest, I am open to new examples.